### ISSUE TITLE
Scoping issue

### ISSUE THREAD

**Issue by mamsincl** [what3words/magento2-w3w-plugin#9]
However, all the configurations are available for all scope levels, the helper responsible for fetching the configured values is asking for values declared for default scope at every time.

simple changing 
https://github.com/what3words/magento2-w3w-plugin/blob/master/Helper/Config.php#L48
to 
```
return $this->scopeConfig
   ->getValue(
       self::PREFIX . $area, 
       ScopeInterface::SCOPE_STORES
   );
``` 

should resolve the scoping and make the solution configurable for even store-view level

**Comment by patruvlad** [comment]
Thanks for your report , this is correct and will be solved ASAP with version 3.0.5

**Comment by mamsincl** [comment]
see 3.0.5
https://github.com/what3words/magento2-w3w-plugin/compare/3.0.4...3.0.5

thanks @patruvlad 

closable
